### PR TITLE
feat(profiling): allow to pass tags

### PR DESCRIPTION
**Pull request by jd**
## feat(profiling): allow to pass tags to the Profiler

### CODE REVIEW

File: ddtrace/profiling/exporter/http.py [hunk 1]
```diff
@@ -97,23 +107,20 @@ def _encode_multipart_formdata(fields, tags):
...
             "service": service.encode("utf-8"),
-            "host": HOSTNAME.encode("utf-8"),
             "runtime-id": runtime.get_runtime_id().encode("ascii"),
-            "language": b"python",
-            "runtime": PYTHON_IMPLEMENTATION,
-            "runtime_version": PYTHON_VERSION,
-            "profiler_version": ddtrace.__version__.encode("utf-8"),
         }
 
+        user_tags = parse_tags_str(os.environ.get("DD_TAGS", {}))
```
Comment (brettlangdon) [inline ddtrace/profiling/exporter/http.py:113]: should we parse these env vars once at creation rather than on every export?

**Review by brettlangdon — APPROVED** [review]

File: ddtrace/profiling/exporter/http.py (hunk 1 above)
Comment (jd) [reply ddtrace/profiling/exporter/http.py:113]: We could, that'd break changing them at runtime. Not sure it's a big deal?

Comment (brettlangdon) [reply ddtrace/profiling/exporter/http.py:113]: Not sure I follow what you mean by changing at runtime?

If the user via code updates the tags or if they modify env variables after the process has started?

Comment (jd) [reply ddtrace/profiling/exporter/http.py:113]: If they modify the env variables after the process has started.

**Comment by jd** [comment]
I'll do a followup with that change.

**Pull request merged**